src/follower.py

```diff
@@ -73,21 +72,53 @@ def stop(self):
 
 
     """
-    Turn the wheels towards the last recognized object.
+    Turn the wheels and pans the camera towards the last recognized object.
     """
     def turn(self):
+        self.pan_camera()
         self.convert_camera_angle()
         self._fw.turn(self._turn_angle)
 
 
+    """
+    Follows the tag by panning camera towards the same direction as the wheels.
+    """
+    def pan_camera(self):
+        if self._tag._tag_data['x'] < 0:
+            self.turn_camera_left()
+        elif self._tag._tag_data['x'] > 0:
+            self.turn_camera_right()
+        else:
+            self.reset_camera() 
+
+
+    """
+    Turn the camera left and right.
+    Turn Left/Right functions from the Camera class only takes Steps.
+    """
+    def turn_camera_left(self):
+        self._camera.turn_left(self.angle_to_steps())
+    
+    def turn_camera_right(self):
+        self._camera.turn_right(self.angle_to_steps())
+
+    
+    """
+    Convert angles to steps.
+    Takes the angle property and converts into steps where 1 step is 15 degrees.
+    """
+    def angle_to_steps(self):
+        steps = np.abs(self._turn_angle/self._camera.PAN_STEP)
```

Comment:
Why do you need the absolute value if both numbers are always positive?

Reply:
This is only because you're calling pan_camera before convert_camera_angle

Reply:
called pan_camera after convert_camera_angle. removed abs